### DISCUSSION TITLE
[v1] How to support both beta and stable versions

### DISCUSSION THREAD

**Discussion by ayangweb** [tauri-apps/tauri#10995] [Q&A]
Hey 👋, what should I do if I want to make it so that users who have beta receiving turned on in v1 receive the beta version, and those who don't have it turned on receive only the stable version?

Get Updates: https://github.com/ayangweb/EcoPaste-Updater/blob/master/api/update.ts

update config: https://github.com/EcoPasteHub/EcoPaste/blob/ba343b2fd836fb60430e39c03d86f21e8f61b730/src-tauri/tauri.conf.json#L118

If my current stable version is v0.1.0, but some users did not update to v0.1.0 in time before the release of v0.1.1-beta.1, then they can't receive the v0.1.0 update. How should I fix this issue please? Thanks!

**Comment by ayangweb** [comment]
@FabianLars Hey 👋, is there any solution to this problem?

**Comment by FabianLars** [comment]
Moved to https://github.com/tauri-apps/tauri/discussions/11069 ...
